Apologies! Still a beginner.

BTW: when I do run `make fmt` I get the following response:

```
go fmt ./...
goimports -w .
make: goimports: No such file or directory
make: *** [fmt] Error 1
````

On `go1.25.3 darwin/arm64`

--- a/no_tui.go
+++ b/no_tui.go
@@ -191,9 +191,9 @@ func renderNoTUI(ctx context.Context, cli *rae.Client, word string) {
 			fmt.Printf("  - %s (%s%s%s)\n", definition.Raw, Bold, definition.Category, Reset)
 		}
 
-    if meaning.Origin != nil && meaning.Origin.Raw != "" {
-      fmt.Printf("\n  %sOrigen:%s %s\n", Bold, Reset, meaning.Origin.Raw)
-    }
+		if meaning.Origin != nil && meaning.Origin.Raw != "" {
+			fmt.Printf("\n  %sOrigen:%s %s\n", Bold, Reset, meaning.Origin.Raw)
+		}
 
 		if meaning.Conjugations != nil {
 			fmt.Printf("\n  %sConjugaciones%s\n", Bold, Reset)

--- a/tui.go
+++ b/tui.go
@@ -249,9 +249,14 @@ func (t *Tui) displayResults(res rae.WordEntry) {
 		}
 
 		if meaning.Origin != nil && meaning.Origin.Raw != "" {
-      t.resultsView.AddItem("", "", 0, nil)
-      t.resultsView.AddItem(fmt.Sprintf("[yellow][::b]Origen:[white] %s", meaning.Origin.Raw), "", 0, nil)
-    }
+			t.resultsView.AddItem("", "", 0, nil)
+			t.resultsView.AddItem(
+				fmt.Sprintf("[yellow][::b]Origen:[white] %s", meaning.Origin.Raw),
+				"",
+				0,
+				nil,
+			)
+		}
 
 		if meaning.Conjugations != nil {
 			t.resultsView.AddItem("", "", 0, nil)

